math/linear_recurrence_matrix.cpp

```diff
@@ -0,0 +1,368 @@
+/**
+ * @brief Evaluate recurrence relation using matrix exponentiation.
+ * @details
+ * Given a recurrence relation; evaluate the value of nth term.
+ * For e.g., For fibonacci series, recurrence series is `f(n) = f(n-1) + f(n-2)`
+ * where `f(0) = 0` and `f(1) = 1`.
+ * Note that the method used only demonstrates
+ * recurrence relation with one variable (n), unlike `nCr` problem, since it has
+ * two (n, r)
+ *
+ * ### Algorithm
+ * This problem can be solved using matrix exponentiation method. For simple
+ * number exponentiation, [see algorithm
```

Comment:
You can use `@see` here (see for example [this algorithm](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/math/gcd_of_n_numbers.cpp#L6)).